grails-app/views/order/_summary.gsp

```diff
@@ -100,7 +103,9 @@
                 </g:elseif>
             </div>
             <div class="button-group">
-                <g:link controller="order" action="addAdjustment" id="${orderInstance?.id}" class="button">
+                <g:link controller="order" action="addAdjustment" id="${orderInstance?.id}" class="button"
+                        disabled="${orderInstance?.status >= OrderStatus.PLACED && !isApprover}"
```

Comment:
This is fine. But since this same logic is being used in a bunch of places we should figure out a way to encapsulate the logic on either Order, User or in a taglib. 
```
<g:if test=${user.canManageAdjustments(order)}">...</g:if>
<g:if test=${order.canManageAdjustments(user)}">...</g:if>
<g:canManageAdjustments order="${orderInstance}">...</g:canManageAdjustments>
```
Or at the very least set a variable within each page to encapsulate. However, I'd prefer one of the other methods since this is copying code in a bunch of places. 
```
<g:set var="isApprover" value="false" />
<g:isUserInRole roles="[org.pih.warehouse.core.RoleType.ROLE_APPROVER]">
    <g:set var="isApprover" value="true" />
</g:isUserInRole>
<g:set var="canManageAdjustments" value="${orderInstance?.status >= OrderStatus.PLACED && !isApprover}"/>
```